#camera: 
#    S: [640, 480] 
#    K: [612.9666137695312, 0.0, 325.5203552246094, 0.0, 611.693115234375, 246.61328125, 0.0, 0.0, 1.0]
#    R: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
#    D: [612.9666137695312, 0.0, 325.5203552246094, 0.0, 0.0, 611.693115234375, 246.61328125, 0.0, 0.0, 0.0, 1.0, 0.0]

camera: 
    S: [1280, 720] 
    K: [1225.93322753906, 0.0, 651.040710449219, 0.0, 917.539672851563, 369.919921875, 0.0, 0.0, 1.0]
    R: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
    D: [1225.93322753906, 0.0, 651.040710449219, 0.0, 0.0, 917.539672851563, 369.919921875, 0.0, 0.0, 0.0, 1.0, 0.0]


tf: # 'X,Y,Z,Qx,Qy,Qz,Qw' for Each sensor.. Converter : https://www.andre-gaschler.com/rotationconverter/
#    robot2camera    : [0.15     , 0,        0.8, 0.5, -0.5, 0.5, -0.5]
    robot2camera    : [0.0 , 0.0, 0.0, 0.5, -0.5, 0.5, -0.5] # Note that sensor height is 0.42!

